tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -1553,9 +1561,15 @@ export const openFileInViewer = async (args: openFileInViewerArgs): Promise<void
         ),
         page.locator(util.format(resourceNameSelector, name)).click()
       ])
-
-      // in case of error <img> doesn't contain src="blob:https://url"
-      expect(await page.locator(previewImage).getAttribute('src')).toContain('blob')
+      const extension = name.split('.').pop()
+      if (extension === 'mp3' || extension === 'ogg') {
+        expect(await page.locator(previewAudio).getAttribute('src')).toContain(name)
+      } else if (extension === 'webm' || extension === 'mp4') {
+        expect(await page.locator(previewVideo).getAttribute('src')).toContain(name)
+      } else {
+        // in case of error <img> doesn't contain src="blob:https://url"
+        expect(await page.locator(previewImage).getAttribute('src')).toContain('blob')
+      }
```

Comment:
could we use switch instead of if-else? 
like this: 
```
switch (extension) {
      case 'mp3':
      case 'ogg':
        expect(await page.locator(previewAudio).getAttribute('src')).toContain(name)
        break
        ..
```